CTM will not build with GEOSchem_GridComp set to develop branch

The CTM will not build when `GEOSchem_GridComp` is checked out to the master branch.

One gets error messages of the following form:
```
/discover/swdev/wjamieso/GEOS/CTM_HEMCO/GEOSctm/src/Components/GEOSctm_GridComp/CTMconvection_GridComp/GenericConvectionMethod_mod.F90(144): error #6632: Keyword arguments are invalid without an explicit interface.   [DEFAULT]
     &               "det_ent:", default=.false., rc=STATUS )
---------------------------------^
/discover/swdev/wjamieso/GEOS/CTM_HEMCO/GEOSctm/src/Components/GEOSctm_GridComp/CTMconvection_GridComp/GenericConvectionMethod_mod.F90(144): error #6632: Keyword arguments are invalid without an explicit interface.   [RC]
     &               "det_ent:", default=.false., rc=STATUS )
--------------------------------------------------^
/discover/swdev/wjamieso/GEOS/CTM_HEMCO/GEOSctm/src/Components/GEOSctm_GridComp/CTMconvection_GridComp/GenericConvectionMethod_mod.F90(148): error #6632: Keyword arguments are invalid without an explicit interface.   [DEFAULT]
     &               "do_downdraft:", default=.false., rc=STATUS )
--------------------------------------^
/discover/swdev/wjamieso/GEOS/CTM_HEMCO/GEOSctm/src/Components/GEOSctm_GridComp/CTMconvection_GridComp/GenericConvectionMethod_mod.F90(148): error #6632: Keyword arguments are invalid without an explicit interface.   [RC]
     &               "do_downdraft:", default=.false., rc=STATUS )
-------------------------------------------------------^
compilation aborted for /discover/swdev/wjamieso/GEOS/CTM_HEMCO/GEOSctm/src/Components/GEOSctm_GridComp/CTMconvection_GridComp/GenericConvectionMethod_mod.F90 (code 1)
src/Components/GEOSctm_GridComp/CTMconvection_GridComp/CMakeFiles/CTMconvection_GridComp.dir/build.make:92: recipe for target 'src/Components/GEOSctm_GridComp/CTMconvection_GridComp/CMakeFiles/CTMconvection_GridComp.dir/GenericConvectionMethod_mod.F90.o' failed
make[2]: *** [src/Components/GEOSctm_GridComp/CTMconvection_GridComp/CMakeFiles/CTMconvection_GridComp.dir/GenericConvectionMethod_mod.F90.o] Error 1
CMakeFiles/Makefile2:9148: recipe for target 'src/Components/GEOSctm_GridComp/CTMconvection_GridComp/CMakeFiles/CTMconvection_GridComp.dir/all' failed
make[1]: *** [src/Components/GEOSctm_GridComp/CTMconvection_GridComp/CMakeFiles/CTMconvection_GridComp.dir/all] Error 2
Makefile:179: recipe for target 'all' failed
make: *** [all] Error 2
```

This is likely related to the following cleanup commit:
 https://github.com/GEOS-ESM/GEOSchem_GridComp/commit/dbcc7fbf2e48fdf4fda4349681234e9af58cad8e#diff-b8425f43f47b5b8b76d182087d4e9d50

It is likely fixed by changing form `rcEsmfReadLogical` to `ESMF_ConfigGetAttribute`.

--- a/src/Components/GEOSctm_GridComp/CTMconvection_GridComp/GenericConvectionMethod_mod.F90
+++ b/src/Components/GEOSctm_GridComp/CTMconvection_GridComp/GenericConvectionMethod_mod.F90
@@ -140,12 +140,12 @@
       VERIFY_(STATUS)
 
 
-      call rcEsmfReadLogical(convConfigFile, self%det_ent, &
-     &               "det_ent:", default=.false., rc=STATUS )
+      call ESMF_ConfigGetAttribute(convConfigFile, value=self%det_ent, &
+                     label="det_ent:", default=.false., rc=STATUS)
       VERIFY_(STATUS)
 
-      call rcEsmfReadLogical(convConfigFile, self%do_downdraft, &
-     &               "do_downdraft:", default=.false., rc=STATUS )
+      call ESMF_ConfigGetAttribute(convConfigFile, value=self%do_downdraft, &
+                     label="do_downdraft:", default=.false., rc=STATUS)
       VERIFY_(STATUS)
 
       IF ( MAPL_AM_I_ROOT() ) THEN

--- a/src/Components/GEOSctm_GridComp/CTMconvection_GridComp/GmiConvectionMethod_mod.F90
+++ b/src/Components/GEOSctm_GridComp/CTMconvection_GridComp/GmiConvectionMethod_mod.F90
@@ -168,16 +168,16 @@
      &                default = 0, rc=STATUS )
       VERIFY_(STATUS)
 
-      call rcEsmfReadLogical(convConfigFile, self%det_ent, &
-     &               "det_ent:", default=.false., rc=STATUS )
+      call ESMF_ConfigGetAttribute(convConfigFile, value=self%det_ent, &
+                     label="det_ent:", default=.false., rc=STATUS)
       VERIFY_(STATUS)
 
-      call rcEsmfReadLogical(convConfigFile, self%do_downdraft, &
-     &               "do_downdraft:", default=.false., rc=STATUS )
+      call ESMF_ConfigGetAttribute(convConfigFile, value=self%do_downdraft, &
+                     label="do_downdraft:", default=.false., rc=STATUS)
       VERIFY_(STATUS)
 
-      call rcEsmfReadLogical(convConfigFile, self%do_old_ncar, &
-     &               "do_old_ncar:", default=.false., rc=STATUS )
+      call ESMF_ConfigGetAttribute(convConfigFile, value=self%do_old_ncar, &
+                     label="do_old_ncar:", default=.false., rc=STATUS)
       VERIFY_(STATUS)
 
       call ESMF_ConfigGetAttribute(convConfigFile, self%met_opt, &
@@ -190,12 +190,12 @@
      &                default = 0, rc=STATUS )
       VERIFY_(STATUS)
 
-      call rcEsmfReadLogical(convConfigFile, self%do_wetdep, &
-     &               "do_wetdep:", default=.false., rc=STATUS )
+      call ESMF_ConfigGetAttribute(convConfigFile, value=self%do_wetdep, &
+                     label="do_wetdep:", default=.false., rc=STATUS)
       VERIFY_(STATUS)
 
-      call rcEsmfReadLogical(convConfigFile, self%do_drydep, &
-     &               "do_drydep:", default=.false., rc=STATUS )
+      call ESMF_ConfigGetAttribute(convConfigFile, value=self%do_drydep, &
+                     label="do_drydep:", default=.false., rc=STATUS)
       VERIFY_(STATUS)
 
       IF ( MAPL_AM_I_ROOT() ) THEN

